nim-mode.el

```diff
@@ -134,7 +134,11 @@
   (setq-local electric-indent-inhibit t)
   (setq-local electric-indent-chars '(?: ?\s))
   (when electric-indent-mode
-    (define-key nim-mode-map [remap delete-backward-char] 'nim-electric-backspace)))
+    (define-key nim-mode-map [remap delete-backward-char] 'nim-electric-backspace))
+
+
+  ;; Set `compile-command' to the correct initial value
+  (nim-compile--set-compile-command))
```

Comment:
I was just following the documentation of ``compile-command`` (``C-h v compile-command``) where it is suggested to set the compile command when entering the major mode.